src/main/java/org/fundacionjala/coding/marcoslara/multiplesofthreeandfive/MultiplesOfThreeAndFive.java

```diff
@@ -0,0 +1,28 @@
+package org.fundacionjala.coding.marcoslara.multiplesofthreeandfive;
+
+/**
+ * Created by Marcos on 12-06-2017.
+ */
+public final class MultiplesOfThreeAndFive {
+
+    /**
+     * Private constructor required. These kind of classes are more like tools/utilities than being meant for objects.
+     */
+    private MultiplesOfThreeAndFive() {
+
+    }
+
+    /**
+     * Main method that calculates the solution of the sum of the multiples of three and five.
+     *
+     * @param number Is the parameter number from which the solution is to be calculated.
+     * @return The sum of the multiples of three and five.
+     */
+    public static int solution(int number) {
+        int sum = 0;
+        for (int i = 3; i <= (number - number % 3); i++) {
```

Comment:
I tried to reduce the number of iterations, but maybe don't work in another scenario. Thanks.